src/components/ErrorNotification.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+interface ErrorNotificationProps {
+  errorMessage: string | null;
+  onHideError: () => void;
+}
+
+export const ErrorNotification: React.FC<ErrorNotificationProps> = ({
+  errorMessage,
+  onHideError,
+}) => {
+  return (
+    <div
+      data-cy="ErrorNotification"
+      className={`notification is-danger is-light has-text-weight-normal ${!errorMessage ? 'hidden' : ''}`}
```

Comment:
```suggestion
      className={`notification is-danger is-light has-text-weight-normal ${errorMessage ? '': 'hidden'}`}
```
MAke conditions as simple as possible